### ISSUE TITLE
License mismatch with PyPI

### ISSUE THREAD

**Issue by mckev-amazon** [TMiguelT/PandasSchema#35]
I happened to notice that the license listed in https://pypi.org/project/pandas-schema/ is listed as MIT, while it is GPLv3 on GitHub.

Just wanted to point that out in case you wanted to fix PyPI so that it matches the license in GitHub (or visa-versa).

**Comment by sureshvigneshbe** [comment]
Yes the same have been noticed , while running pip-licenses , it return MIT and not GPL.
